cmd/cantool/main.go

```diff
@@ -51,6 +51,9 @@ func generateCommand(app *kingpin.Application) {
 		Arg("output-dir", "output directory").
 		Required().
 		String()
+	whiteList := command.
+		Arg("message-ids", "optional message-id whitelist").
+		Uint32List()
```

Comment:
Perhaps call this option something more descriptive, as `only-compile` or something similar, both to be more descriptive, but also being more inclusive (i.e. avoid the term whitelist in the code base).

Reply:
Good point, i decided to go with "allowed message ids". Let me know what you think